src/load_inputs/load_network_data.jl

```diff
@@ -80,6 +80,17 @@ function load_network_data!(setup::Dict, path::AbstractString, inputs_nw::Dict)
         inputs_nw["NO_EXPANSION_LINES"] = findall(inputs_nw["pMax_Line_Reinforcement"].<0)
     end
 
+    ## Inputs for the DC-OPF 
+    if setup["DC_OPF"] == 1
+        println("Reading DC-OPF values...")
+        # Transmission line voltage (in kV)
+        inputs_nw["kV"] = to_floats(:Line_Voltage_kV)
+        # Transmission line impedance (in Ohms)
+        inputs_nw["Impedance_Ohms"] = to_floats(:Line_Impedance_ohms)    
+        inputs_nw["LINE_Angle_Limit"] = to_floats(:Angle_Limit_Deg)
+        inputs_nw["pDC_OPF_coeff"] = ((inputs_nw["kV"]/10^3).*(inputs_nw["kV"]/10^3))./(inputs_nw["Impedance_Ohms"]/10^6) * scale_factor # 1/GW ***
```

Comment:
I'd suggest that rather than four scale or conversion factors in this expression, just have one numerical factor and the one scale factor. 

Name the fixed numerical factor something like 

```
kilovolt_to_megavolt = 10^-3
```
.
In general it's bad practice to have un-named numerical constants, because you don't always know what they represent.